src/content/docs/data-apis/understand-data/new-relic-data-types.mdx

```diff
@@ -1,11 +1,11 @@
 ---
-title: New Relic data types
+title: New Relic MELT data types
```

Comment:
I was told previously that we shouldn't use MELT like this in public docs, that it was more of an occasional marketing thing, and some people seemed conflicted on whether it even made sense marketing-wise. did that change? making sure. if so, should we be promoting "MELT" more widely?